src/Header.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { NavLink } from 'react-router-dom';
+class Header extends Component {
+    
+    render() { 
+        return ( 
+            <header>
+                <h2>Here the header</h2>
+                <div className='links'>
+
+                    <NavLink exact to='/'>
+                        Home
+                    </NavLink>
+                    <NavLink exact to='/pokemon'>
+                        Pokemon List
+                    </NavLink>
+                    <NavLink exact to='/pokemon-details'>
```

Comment:
You don't want to link to your details from the header here, you'll want to make this link from each individual pokemon item from your PokeItem component so it knows which pokemon to give the details for (from the id in params).